pkg/debuginfo/metadata.go

```diff
@@ -96,98 +98,120 @@ func newMetadataManager(logger log.Logger, bucket objstore.Bucket) *metadataMana
 
 type metadata struct {
 	State            metadataState `json:"state"`
-	StartedUploadAt  int64         `json:"started_upload_at"`
-	FinishedUploadAt int64         `json:"finished_upload_at"`
+	BuildID          string        `json:"build_id"`
```

Comment:
Why is this needed? The BuildID is already in the path of the object, no?

Reply:
I think it would be convenient to have the build_id here, in general. Especially since it's typed and not a loose string that might change in the future and might make use add awkward build_id parsing out of the path

Reply:
I don't feel strongly. I'm ok with keeping it.